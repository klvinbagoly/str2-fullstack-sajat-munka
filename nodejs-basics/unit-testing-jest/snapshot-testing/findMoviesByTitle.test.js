const findMoviesByTitle = require('./findMoviesByTitle')

test('findMoviesByTitle should return an array of movie objects', () => {
  // expect(findMoviesByTitle('F')).toEqual([{ title: 'Flipper', emoji: '🐬' }])
  expect.addSnapshotSerializer({
    test: ({ title, emoji }) => title && emoji,
    print: ({ title, emoji }) => `${emoji} ${title}`
  })
  expect(findMoviesByTitle('F')).toMatchSnapshot()
})
