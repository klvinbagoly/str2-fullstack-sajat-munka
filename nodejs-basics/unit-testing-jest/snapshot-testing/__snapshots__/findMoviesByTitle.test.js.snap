// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`findMoviesByTitle should return an array of movie objects 1`] = `
Array [
  🐬 Flipper,
]
`;
